openid-caep-interoperability-profile-1_0.md

```diff
@@ -175,6 +203,36 @@ Receivers MUST be prepared to accept events with any of the subject identifier f
 ## Event Signatures
 All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
 
+## OAuth Service
+
+### Authorization Server
+* MAY distribute discovery metadata (such as the authorization endpoint) via the metadata document as specified in [RFC8414]{{RFC8414}}
+* MUST support at least one of the following to obtain a short-lived access token. A short lived access token could be defined as `exp` claim no longer than 60 mins after `nbf`. Please refer Access token lifetimes in the security considerations {{FAPI}} for additional considerations. 
+** client credential grant flow {{RFC6749}} section 4.4
+** authorization code flow {{RFC6749}} section 4.1
+
+### OAuth Scopes
+The client could discover OAuth scopes needed to access the reource by 2 separate mechanisms depending on the support from the OAuth service
+
+#### Resource Server
+The OAuth scopes required to access the resources MAY be obtained by the client using {{OPRM}} hosted by the Resource Server. 
```

Comment:
The Resource Server here is the SSF API, right? Why not just say that? Also, these two sections could be specified as a process i.e. The client first does X, but if not then Y.